eth-contracts/contracts/Governance.sol

```diff
@@ -691,4 +696,18 @@ contract Governance is InitializableV2 {
             proposals[_proposalId].voteMagnitudeNo.sub(_voterStake)
         );
     }
+
+    /**
+     * @notice Returns true if voting quorum percentage met for proposal, else false.
+     * @dev Quorum is met if total voteMagnitude * 100 / total active stake in Staking
```

Comment:
Can we add a comment here showing the calculation for when this would have overflow issues assuming 100% staking participation - log(2^256/10^29)/log(1.07) = 1635 years at 7% PA inflation

Reply:
added